crates/anvil/src/eth/backend/mem/mod.rs

```diff
@@ -1114,7 +1114,18 @@ impl Backend {
         block_env: BlockEnv,
     ) -> EnvWithHandlerCfg {
         let WithOtherFields::<TransactionRequest> {
-            inner: TransactionRequest { from, to, gas, value, input, nonce, access_list, .. },
+            inner:
```

Comment:
something we could consider here is taking the unused fields into `_` instead of using `..` to require us to figure out what to do with new fields when we add them

e.g.

```rs
inner: TransactionRequest {
  from,
  to,
  // ...
  max_fee_per_blob_gas: _,
  some_other_field: _,
}